packages/client/src/built-in/setting-layout/SettingLayout.tsx

```
@@ -20,17 +20,24 @@ export const SettingLayout = ({ selectedKeys, onClick, route, children, fullscre
     fixedHeader: true,
     siderMenuType: 'sub',
   };
-
+  const app = useApp();
   const location = useLocation();
   const navigate = useNavigate();
   const { data } = useCurrentUserContext();
   const result = useSystemSettings();
+  const { styles } = useStyles();
+  const redirectToUserHome = () => (window.location.href = app.adminUrl);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)